src/main/java/strategy/handler/BalancesHandlerImpl.java

```diff
@@ -0,0 +1,27 @@
+package strategy.handler;
+
+import dao.WokWithStorageImpl;
+import dao.WorkWithStorageDB;
+import model.FruitTransaction;
+
+public class BalancesHandlerImpl implements Handler {
+    private WorkWithStorageDB storageService = new WokWithStorageImpl();
+
+    public void handler(FruitTransaction fruitTransaction) {
+        if (fruitTransaction.getQuantity() <= 0) {
+            throw new RuntimeException("Can't work with this quantity "
+                    + fruitTransaction.getQuantity());
+        }
```

Comment:
let's create a validation service that will check if a transaction is valid, so you don't need to duplicate this logic for every handler